job-dsls/src/main/groovy/org/kie/jenkins/jobdsl/Constants.groovy

```diff
@@ -51,6 +51,7 @@ class Constants {
     static final String KERBEROS_CRED = ''
     static final String IMAGE_REPO = ''
     static final String IMAGE_BRANCH = ''
+    static final String BAMOE_IMAGE_BRANCH = '7.13.x-blue'
```

Comment:
@Ginxo I've got a question: why don't we need to set the IMAGE_BRANCH constant that is used for the IMAGE_BRANCH string param on the RHPAM images? Where are we retrieving this param from tor the non BAMOE components?

Reply:
it is retrieved in the download.sh using the PROD_VERSION.

Reply:
So we could add a BAMOE_IMAGE_BRANCH_EXT = '-blue' and append it to the IMAGE_BRANCH in order not to have to change it and have it automatically aligned, even if we won't need anything different from the current value.
Do you think that this change is worth?

Reply:
it is not really needed, I've changed the download.sh script as well.
we can keep this new env or delete.